Foundation/include/Poco/FileStream_POSIX.h

```diff
@@ -61,7 +61,7 @@ class Foundation_API FileStreamBuf: public BufferedBidirectionalStreamBuf
 	NativeHandle nativeHandle() const;
 		/// Returns native file descriptor handle
 	
-	Poco::UInt64 size() const;
+	UIntPtr size() const;
```

Comment:
Why changing the return type?

Reply:
Result shoud be platform depended because we use it for sendfile.

Reply:
I have reverted